estrategiacontenidos/mapadelsitio.md

```diff
@@ -1,16 +1,21 @@
 # Mapa del sitio
 
-<!--
-Recoged el mapa del sitio en este documento en forma de lista
-formateada usando la sintaxis de Markdown, así:
-
-- si queréis
-- una lista
-  + sin ordenar
-
-o así:
-
-1. si queréis
-   1. una lista
-   2. ordenada
--->
+1. Página principal
+    1. **Podcast**
+         1. Noticiario 1
+             - Audio noticiario 1
+         2. Noticiario 2
+             - Audio noticiario 2
+         3. Noticiario 3
+              - Audio noticiario 3
+         4. Tomas Falsas
+              - Lista de Vídeos
+    2. **Revista**
+         1. "Política y Pipas"
+             - Audios de las entrevistas
+    3. **Reportaje de TV**
+          1. (Título del reportaje)
```

Comment:
¿Una sección por reportaje? ¿O sólo hay previsto uno?

Reply:
Sí, solo hay un reportaje audiovisual que va con el tema porque el resto de los tratan sobre cuestiones distintas.

Reply:
Solo hay uno **ahora**, ¿pero imagináis que en el futuro siempre seguirá habiendo sólo uno?

Reply:
Se añadirán más pero aún no tenemos contenido para poder mencionar una sección en la que no poner nada